iohk-monitoring/src/Cardano/BM/Backend/LogBuffer.lhs

```diff
@@ -69,10 +69,10 @@ Function |effectuate| is called to pass in a |LogObject| for log buffering.
 instance IsEffectuator LogBuffer a where
     effectuate buffer lo@(LogObject loname _lometa (LogValue lvname _lvalue)) =
         modifyMVar_ (getLogBuf buffer) $ \currentBuffer ->
-            return $! LogBufferInternal $ HM.insert ("#buffered." <> loname2text loname <> "." <> lvname) lo $ logBuffer currentBuffer
+            return $! LogBufferInternal $ HM.insert ("#buffered." <> loname <> "." <> lvname) lo $ logBuffer currentBuffer
```

Comment:
There is plenty of string concatenation. If the list is getting large, you can use:
```
$> concat ["test", " ", "this"]
"test this"
```